components/ReferenceManager/references/reference_organizer/ReferenceProjectsUpsertModal.tsx

```diff
@@ -65,8 +65,14 @@ export default function ReferenceProjectsUpsertModal({
       is_public: isPublic,
       organization: currentOrg?.id,
       project_name: nullthrows(projectName, "Project name may not be null"),
-      project: projectID,
     };
+
+    if (upsertPurpose === "update") {
+      formattedPayload.project = projectID;
+    } else {
+      formattedPayload.parent = projectID;
+    }
```

Comment:
Are we confident that creating a project will always have an active current project?
Not a fan of if statements like this that mutate already existing objects either. 
Can we do something like what i suggested in above?